addons/sourcemod/scripting/shop/admin.sp

```diff
@@ -74,6 +74,18 @@ public int Admin_ShowAdminMenu(Handle plugin, int numParams)
 	Admin_ShowMenu(client);
 }
 
+void Admin_UnregisterMe(Handle hPlugin)
+{
+	int index = -1;
+	while ((index = g_hAdminArray.FindValue(hPlugin)) != -1)
+	{
+		delete view_as<Handle>(g_hAdminArray.Get(index+1));
+
+		g_hAdminArray.Erase(index);
+		g_hAdminArray.Erase(index);
+	}
+}
+
```

Comment:
It can be replaced by public function.
```hpp
public int Native_UnregisterMe(Handle plugin, int params)
{
    // ....
      Admin_RemoveFromMenuNative(plugin, params);
}
```